aten/src/ATen/native/native_functions.yaml

```diff
@@ -4307,6 +4315,7 @@
     CUDA: batch_norm_cuda
     MPS: batch_norm_mps
     MkldnnCPU: mkldnn_batch_norm
+  tags: maybe_aliasing_or_mutating
```

Comment:
I'm not sure about this, in these cases the schema is just wrong (as opposed to composites, where schema is not meaningful)

Reply:
Do you mean we don't need this tag at all or we need two seperate tags (one for CIA ops that are maybe mutating and one for primitive ops)?